tests/integration/gha-run.sh

```diff
@@ -0,0 +1,153 @@
+#!/usr/bin/env bash
+
+# Copyright (c) 2023 Microsoft Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+integration_dir="$(dirname "$(readlink -f "$0")")"
+tools_dir="${integration_dir}/../../tools"
+
+function _print_cluster_name() {
+    short_sha="$(echo "${GH_PR_SHA}" | cut -c -12)"
```

Comment:
Fabiano recommended that we use `git rev-parse --short=12 HEAD` instead of string manipulation.